### ISSUE TITLE
Display image descriptions even without mouseover

### ISSUE THREAD

**Issue by aarchiba** [BeachCity/mastodon#71]
Currently it is cumbersome to see image descriptions even when one has clicked on an image to expand it - one must mouse over the image in the viewer, pause, then the mouseover text appears, concealing some part of the image. This discourages users from looking at image descriptions unless they need them. In the interest of establishing useful image descriptions as a norm, I would like to suggest making image descriptions always visible in the image viewer.

Specifically, there are three ways images are shown in the web viewer:
1. Packed into the toot's own display. Here they are cropped and/or shrunk. Currently image descriptions appear on mouseover. (Good: they don't take up space and allow users to look at the image description to judge whether to click to enlarge the image.)
2. After clicking on an image in the toot. The image pops up modally, appearing in the center of the user's screen. A rudimentary viewer is available, including an X to close the viewer, right and left arrows if there are multiple images, and a series of dots to indicate which of multiple images is being shown. Currently image descriptions appear on mouseover of the large image. In this issue I propose making the image description appear, in the same format as the mouseover text (small, white text in a dark gray box), at the bottom right (for example) of the screen. This should not conceal any part of the image and it should make the presence and contents of the image description apparent with no user effort.
3. After clicking on an image in the viewer. Other user interface elements fade out and only the image is visible floating over the usual mastodon window. Clicking on the image again in this state returns the rudimentary viewer; clicking outside the image exits image viewing. Currently the image description is not visible in this state. This state exists to hide unwanted UI elements, and I think it would serve to hide the image description display I am proposing should it obscure part of an image (perhaps the description is very large or the image in the viewer fills the window). 

In short: I propose replacing the mouseover in viewer number 2 above with a permanent display in a corner of the screen. Should this obscure part of the image users can use the above described viewer 3 to make it disappear, but because of the way mastodon images are scaled for display in viewers 2 and 3 I do not expect this to happen often.
